lib/helper/Testcafe.js

```diff
@@ -0,0 +1,661 @@
+   */
+  async resizeWindow(width, height) {
+    if (width === 'maximize') {
+      return this.t.maximizeWindow().catch(mapError);
+    }
+
+    return this.t.resizeWindow(width, height).catch(mapError);
+  }
+
+  /**
+   * {{> click }}
+   *
+   * {{ react }}
+   */
+  async click(locator, context = null) {
+    return proceedClick.call(this, locator, context);
+  }
+
+
+  /**
+   * {{> refreshPage }}
+   */
+  async refreshPage() {
+    // eslint-disable-next-line no-restricted-globals
+    return this.t.eval(() => location.reload(true), { boundTestRun: this.t }).catch(mapError);
+  }
+
+  /**
+   * {{> waitForVisible }}
+   *
+   * This method accepts [React selectors](https://codecept.io/react).
+   */
+  async waitForVisible(locator, sec) {
+    const timeout = sec ? sec * 1000 : undefined;
+
+    return findElements.call(this, this.context, locator)
+      .with({ visibilityCheck: true, timeout })()
+      .catch(mapError);
+  }
+
+  /**
+   * {{> fillField }}
+   * {{ react }}
+   */
+  async fillField(field, value) {
+    return this.t
+      .typeText(findElements.call(this, this.context, field), value, { replace: true })
+      .catch(mapError);
+  }
+
+  /**
+   * {{> pressKey }}
+   *
+   * {{ keys }}
+   */
+  async pressKey(key) {
+    assert(key, 'Expected a sequence of keys or key combinations');
+
+    return this.t
+      .pressKey(key.toLowerCase()) // testcafe keys are lowercase
+      .catch(mapError);
+  }
+
+  /**
+   * {{> hover }}
+   *
+   * {{ locator }}
+   */
+  async hover(locator) {
+    return this.t
+      .hover(findElements.call(this, this.context, locator))
+      .catch(mapError);
+  }
+
+  /**
+   * {{> doubleClick }}
+   *
+   * {{ react }}
+   */
+  async doubleClick(locator, context = null) {
+    return this.t
+      .doubleClick(findElements.call(this, this.context, locator))
+      .catch(mapError);
+  }
+
+
+  /**
+   * {{> seeInCurrentUrl }}
+   */
+  async seeInCurrentUrl(url) {
+    stringIncludes('url').assert(url, await getPageUrl(this.t)().catch(mapError));
+  }
+
+  /**
+    * {{> dontSeeInCurrentUrl }}
+    */
+  async dontSeeInCurrentUrl(url) {
+    stringIncludes('url').negate(url, await getPageUrl(this.t)().catch(mapError));
+  }
+
+  /**
+    * {{> seeCurrentUrlEquals }}
+    */
+  async seeCurrentUrlEquals(url) {
+    urlEquals(this.options.url).assert(url, await getPageUrl(this.t)().catch(mapError));
+  }
+
+  /**
+    * {{> dontSeeCurrentUrlEquals }}
+    */
+  async dontSeeCurrentUrlEquals(url) {
+    urlEquals(this.options.url).negate(url, await getPageUrl(this.t)().catch(mapError));
+  }
+
+  /**
+   * {{> see }}
+   *
+   * {{ react }}
+   */
+  async see(text, context = null) {
+    let els;
+    if (context) {
+      els = findElements.call(this, this.context, context)
+        .withText(text);
+    } else {
+      // TODO Check how that is implemented in Puppeteer
+      els = findElements.call(this, this.context, `//*[contains(text(),'${text}')]`);
```

Comment:
The text should be escaped to be passed into XPath. Maybe you can have `new Locator().withText(text)` ?